pyproject.toml

```
@@ -39,7 +39,10 @@ classifiers = [
   "Topic :: Software Development :: Libraries :: Python Modules",
   "Topic :: Software Development :: Quality Assurance",
 ]
-urls = { repository = "https://github.com/charliermarsh/ruff" }
+
+[project.urls]
+Repository = "https://github.com/charliermarsh/ruff"
+Changelog = "https://github.com/charliermarsh/ruff/releases"
```

Perhaps also add docs link

```suggestion
Documentation = "https://beta.ruff.rs/docs/"
Changelog = "https://github.com/charliermarsh/ruff/releases"
```